src/include/cpu/power/istep_18.h

```diff
@@ -0,0 +1,634 @@
+// [27]    RW PRI_X7_PORT_0_TX_ENABLE: Distribution: primary configuration: X7 port-0 TX enable.
+//         0 = Port configured as receiver.
+//         1 = Port configured as sender.
+// [28:31] RW REG_0X01_SPARE_28_31: Spares.
+// [32:39] RW PRI_I_PATH_DELAY_VALUE: Internal path: primary configuration: delay value.
+// [40:63] RO constant = 0
+#define PERV_TOD_PRI_PORT_0_CTRL_REG (0x00040001)
+// TOD: Slave Path Control Register
+// [0]     RW PRI_S_PATH_SELECT: Primary configuration: slave path select.
+// [1]     RW REG_0X05_SPARE_01: Spares.
+// [2]     RW S_PATH_M_PATH_CPS_ENABLE: Slave path-01: use of master path CPS: enable.
+//         0 = Do not use master path CPS.
+//         1 = Use master path CPS.
+// [3]     RW S_PATH_REMOTE_SYNC_DISABLE: Slave path-01: remote sync: disable.
+//         0 = Use Syncs from master TOD. Steps are generated locally.
+//         1 = Use steps-syncs from master TOD.
+// [4]     RW SEC_S_PATH_SELECT: Secondary configuration: slave path select.
+// [5]     RW REG_0X05_SPARE_05: Spares.
+// [6:7]   RW S_PATH_STEP_CHECK_CPS_DEVIATION_FACTOR: Slave path-01: step check: CPS deviation factor.
+// [8:11]  RW S_PATH_0_STEP_CHECK_CPS_DEVIATI1 = Slave path-0: step check: CPS deviation.
+// [12]    RW S_PATH_0_STEP_CHECK_CONSTANT_CPS_ENABLE: Slave path-0: step check: constant CPS enable
+//         0 = Measured CPS is used for the step check CPS deviation
+//         1 = Constant CPS is used for the step check CPS deviation
+// [13:15] RW S_PATH_0_STEP_CHECK_VALIDITY_COUNT: Slave path-0: step check: validity count. Defines the number of received steps before the step is declared as valid is enabled.
+// [16:19] RW S_PATH_1_STEP_CHECK_CPS_DEVIATION: Slave path-1: Step check: CPS deviation.
+// [20]    RW S_PATH_1_STEP_CHECK_CONSTANT_CPS_ENABLE: Slave path-1: Step check: constant CPS enable
+//         0 = Measured CPS is used for the step check CPS deviation.
+//         1 = Constant CPS is used for the step check CPS deviation.
+// [21:23] RW S_PATH_1_STEP_CHECK_VALIDITY_COUNT: Slave path-1: Step check: validity count. Defines the number of received steps before the step is declared as valid.
+// [24]    RW S_PATH_REMOTE_SYNC_ERROR_DISABLE: Slave path-01: Remote sync: Error disable.
+//         0 = Enable remote sync error as a step error.
+//         1 = Do not enable remote sync error as a step error.
+// [25]    RW S_PATH_REMOTE_SYNC_CHECK_M_CPS_DISABLE: Slave path-01: Remote sync: sync-step check: disable use of master path CPS.
+//         0 = Use master path CPS.
+//         1 = Do not use master path CPS.
+// [26:27] RW S_PATH_REMOTE_SYNC_CHECK_CPS_DEVIATION_FACTOR: Slave path-01: remote sync: sync-step check: CPS deviation factor.
+// [28:31] RW S_PATH_REMOTE_SYNC_CHECK_CPS_DEVIATION: Slave path-01: remote sync: sync-step check: CPS deviation.
+// [32:39] RW S_PATH_REMOTE_SYNC_MISS_COUNT_MAX: Slave path-01: remote sync: maximum of SYNC miss counts: 0 - 255 syncs.
+// [40:63] RO constant = 0
```

Comment:
This comment applies to whole file.

This is not the documentation, it should be enough to mention only the bits that are actually used. Sometimes just a name of the bit in the form of macro is self-explanatory. The documentation is publicly available, there is no need to copy it. Right now this file has ~600 lines that no one will ever need to read.

Reply:
Removed most of the comments, leaving only information about used bit fields.